app/services/flow/flows/provider_merits.rb

```diff
@@ -124,6 +124,10 @@ class ProviderMerits < FlowSteps # rubocop:disable Metrics/ClassLength
         },
         merits_task_lists: {
           path: ->(application) { urls.providers_legal_aid_application_merits_task_list_path(application) },
+          forward: :gateway_evidences
+        },
```

Comment:
From reading the ticket, gateway evidence is only required if multiple proceedings have been selected, so I think here we want to go straight on to check merits answers if there is only one proceeding.